.github/workflows/cd.yaml

```diff
@@ -0,0 +1,35 @@
+name: cd
+on:
+  pull_request:
+    branches:
+      - master
+    types: [closed]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-18.04
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.AWS_REGION }}
+      - name: package
+        run: npm run predeploy
+      - name: upload
+        if: ${{ steps.package.conclusion == 'success' }}
```

Comment:
Are these checks necessary? I've always seen GitHub actions run sequentially and bail out if a step fails